### ISSUE TITLE
Non-linear activation for visibility values

### ISSUE THREAD

**Issue by iancze** [MPoL-dev/MPoL#40]
**Is your feature request related to a problem or opportunity? Please describe.**
Poorly calibrated visibilities (including the visibility *weights*) may negatively impact the image solutions. This could be due to a number of reasons, but probably the main one is that a noise-dominated measurement is taken at face value in the likelihood function and pulls the image in an unwanted direction.

We most likely have priors for how visibilities should behave as a group of antenna, time, uv distance etc. So provide a tunable parameter in the model (per visibility sample, per antenna, etc...) that acts sort of like a bit-flip on what amplitude the visibility should enter the likelihood calculation (and with what weight). 

**Describe the solution you'd like**
This could be something like connecting the loose visibilities through something like [Sigmoid](https://pytorch.org/docs/stable/generated/torch.nn.Sigmoid.html#torch.nn.Sigmoid) that multiplies the visibility value.

Most likely we'll want this to work on the loose visibilities directly, but it could probably also be implemented on GriddedDatasets.

Then, the weight is determined at a later stage of the training process.

**Additional context**
It's possible we could end up in some weird parameter spaces, which is always a risk when you start modifying data values. The trick will be identifying priors and training approaches that minimize pathological solutions (e.g., like *all* visibilities set to zero).

We could also try "robust" likelihood functions like Student-t or Lorentzians instead of the chi^2 likelihood function we have currently. 

Regardless of visibility activations, it is very important to make sure that the weights are scaled correctly (#37).

**Comment by iancze** [comment]
Closing this for now, since we seem to understand the data calibration a bit better (see [visread walkthroughs](https://mpol-dev.github.io/visread/)).
